src/components/KIcon/KIcon.vue

```diff
@@ -28,20 +28,21 @@
 <script lang="ts" setup>
 import { ref, computed, onMounted, nextTick, watch, useAttrs, useSlots } from 'vue'
 import * as allIcons from './icons'
+import { KUI_ICON_SIZE_50 } from '@kong/design-tokens'
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 const icons: Record<string, any> = allIcons
 
 const DEFAULTS = {
-  size: '24',
+  size: KUI_ICON_SIZE_50,
```

Comment:
This size token is appended with `px` - will that still work with the usage here?

Essentially changing the default from `'24'` to `'24px'`

Reply:
Seems so https://github.com/Kong/design-tokens/blob/main/TOKENS.md?plain=1#L1291

Reply:
Also https://github.com/Kong/kongponents/pull/1558/files#r1270942493